      build_and_train_models/sm-distributed_model_parallel_v2/shared-scripts/arguments.py

```diff
@@ -206,6 +206,11 @@ def parse_args():  # pylint: disable=too-many-statements
     )
     model_grp.add_argument("--rotary_pct", type=float, default=0.25)
     model_grp.add_argument("--rotary_emb_base", type=int, default=10000)
+    model_grp.add_argument("--rope_scaling_type", type=str, choices=["llama3"], default=None)
```

Comment:
Nit: Can you descriptions for what these args do?

Reply:
Thanks! Updated